_release-pypi/action.yml

```diff
@@ -52,6 +52,11 @@ inputs:
     required: true
     type: boolean
 
+  skip-existing:
+    description: >
+      Continue uploading files if one already exists. Only valid when uploading
+      to PyPI. Other implementations may not support this.
+
```

Comment:
Does this need a required or type value like the previous inputs?

Reply:
Yes it does. Please add it as a boolean @jorgepiloto